eng/pipelines/libraries/helix-queues-setup.yml

```
@@ -103,7 +103,7 @@ jobs:
 
     # Android
     - ${{ if in(parameters.platform, 'android_x86', 'android_x64', 'linux_bionic_x64') }}:
-      - Ubuntu.1804.Amd64.Android.29.Open
+      - Ubuntu.2204.Amd64.Android.29.Open
```

One more android case that I missed.

```suggestion
      - Ubuntu.1804.Amd64.Android.29.Open
```

---

Yeah, I was kinda surprised that was working, which is why I queued extra-plat again

---

I'll wait to commit that until the full run is complete. I think that's what happened last time. I think I took the other suggestion while the other run was running and maybe that stopped it?